docs/spec/v1beta3/providers.md

```diff
@@ -1474,6 +1474,34 @@ spec:
       name: flux-system
 ```
 
+##### Construction of Git Commit Status ID
+
+The commit status ID is constructed through the `eventSources.kind`, `eventSources.name` and the 
+[`UID`](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids) of the `Provider`.
+
+To generate a speakable prefix, instead of relying on the `UID` of the `Provider`, `eventMetadata` can be set.
+
+```yaml
+apiVersion: notification.toolkit.fluxcd.io/v1beta3
+kind: Alert
+metadata:
+  name: github-status
+  namespace: flux-system
+spec:
+  providerRef:
+    name: github-status
+  eventMetadata:
+    app.kubernetes.io/env: "production"
+    app.kubernetes.io/cluster: "my-cluster"
+    app.kubernetes.io/region: "us-east-1"
```

Comment:
I'm not sure if the `app.kubernetes.io` is supposed to be used here. My interpretation has been that it's exclusive to well-known labels defined by Kubernetes.

Reply:
I would count it to be okay as it is already in the [alerts.md](https://github.com/fluxcd/notification-controller/blob/main/docs/spec/v1beta3/alerts.md#example-1) mentioned

Reply:
👌🏻